ch.threema.threema-desktop.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <metadata_license>CC0-1.0</metadata_license>
+  <id>ch.threema.threema-desktop</id>
+  <name>Threema</name>
+  <project_license>AGPL-3.0</project_license>
+  <summary>Threema is more than a chat app. Threema is a statement. A statement for privacy.</summary>
+  <url type="homepage">https://threema.ch/</url>
+  <url type="help">https://threema.ch/support</url>
+  <description>
+    <p>Threema for desktop is a desktop client for Threema, a privacy-focused
+    end-to-end encrypted mobile messenger hosted and developed in
+    Switzerland. With Threema for desktop, you can use Threema on your
+    desktop without compromising security.</p>
+  </description>
+  <releases>
+    <release version="1.0.3" date="2021-10-26"/>
+  </releases>
+  <launchable type="desktop-id">ch.threema.threema-desktop.desktop</launchable>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-audio">intense</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+  </content_rating>
```

Comment:
it's missing screenshots.

Reply:
Is it recommended to create your own or link to screenshots from the official website?

Reply:
It probably doesn't matter for Electron apps as they are fairly uniform on all platforms.

Reply:
I accidentally deleted my answer:  I was more concerned wrt privacy and security, as this would lead the software center to access third-party servers (tracking) and possible "parse" the downloaded images.  Further, the website may be updated outside of our control. OTOH, the websites screenshots are often nice because they contain example conversations which I cannot do locally that easily.

Reply:
I've added one screenshot now and will add more later